server/routes/appointments/appointmentsController.ts

```diff
@@ -287,7 +291,23 @@ export default class AppointmentsController {
         formError = data.error
         userInputData = req.body
       } else {
-        await this.draftsService.updateDraft(draft.id, data.paramsForUpdate, { userId: res.locals.user.userId })
+        const draftAppointment: DraftAppointment = {
+          ...data.paramsForUpdate,
+          sessionFeedback: {
```

Comment:
we shouldn't be setting this here - this is because it's possible for a user to be on `check-your-answers` page before submitting and it will direct them to attendance page but the draft won't have an empty sessionFeedback.

We should instead be doing this on the attendance page.